src/GithubMergeTool/GithubMergeTool.cs

```diff
@@ -342,6 +357,12 @@ Task<HttpResponseMessage> AddAssignees(string prNumber, List<string> assignees)
                 // https://developer.github.com/v3/issues/assignees/#add-assignees-to-an-issue
                 return _client.PostAsyncAsJson($"repos/{repoOwner}/{repoName}/issues/{prNumber}/assignees", JsonConvert.SerializeObject(new { assignees }));
             }
+
+            Task<HttpResponseMessage> CheckAssignee(string assignee)
```

Comment:
Maybe move the StatusCode check inside this method. Named something like `IsAssigneeValidAsync`.